bin/create_node_symlink.php

```diff
@@ -1,19 +1,45 @@
 <?php
```

Comment:
Why these changes?

Reply:
https://github.com/Sylius/PluginSkeleton/pull/321/files

Reply:
It is now aligned with the Sylius plugin skeleton. I usually check for also not differed files to be sure that the plugin is maintained in sync with the skeleton.